examples/jmespath_functions/src/extract_data_from_builtin_envelope.py

```diff
@@ -0,0 +1,9 @@
+from aws_lambda_powertools.utilities.jmespath_utils import envelopes, extract_data_from_envelope
+from aws_lambda_powertools.utilities.typing import LambdaContext
+
+
+def handler(event: dict, context: LambdaContext) -> str:
+    payload = extract_data_from_envelope(data=event, envelope=envelopes.SQS)
```

Comment:
Changed the code to use SQS envelope instead SNS envelope and avoid errors using the payload.